firmware/Makefile

```diff
@@ -6,14 +6,21 @@ default: build
 clean: 
 	rm -rf $(APPLICATION_ROOT)/$(OUTPUT)
 
-build:
-	@echo "Compiling SignBuddy..."
-	@./tools/compile.sh
+nucleo:
+	@echo "Compiling SignBudy for nucleo..."
+	@./tools/compile.sh NUCLEO
+
+disco:
+	@echo "Compiling SignBuddy for discovery..."
+	@./tools/compile.sh DISCO
 
 format:
 	@echo "Formatting Source Code..."
 	@./tools/run_formatter.sh
 
-flash:
-	@make -C $(APPLICATION_ROOT) flash
+flash_disco:
+	@make -C $(APPLICATION_ROOT) flash BOARD_VERSION=DISCO
+
+flash_nucleo:
+	@make -C $(APPLICATION_ROOT) flash BOARD_VERSION=NUCLEO
 
```

Comment:
@stephen99scott changed this stuff a bit lmk what you think

Reply:
Looks good